packages/server/src/modules/reports/typeDefs/dynamic-report.graphql.ts

```
@@ -0,0 +1,46 @@
+
+  " a single node of dynamic report template "
+  type DynamicReportNode {
+    id: String!
+    parent: String!
+    text: String!
+    droppable: Boolean!
+    data: DynamicReportNodeData!
+  }
+
+  " data of a single node of dynamic report template "
+  type DynamicReportNodeData {
```

ESLint

[error] type "DynamicReportNodeData" must have exactly one non-nullable unique identifier.
Accepted name: id.
Accepted types: ID or UUID.

(@graphql-eslint/strict-id-in-types)